app/services/facilitator_service.js

```diff
@@ -0,0 +1,17 @@
+import realm from '../db/schema';
+
+const facilitatorService = (() => {
+  return {
+    deleteFacilitators,
+  }
+
+  function deleteFacilitators(scorecardUuid) {
```

Comment:
@limkimsan consider to refactor delete method into a BaseModelService then extends each service to reuse this base method(find, create, update, delete) from this BaseModelService